pyexasol/connection.py

```diff
@@ -38,21 +38,38 @@ class Host(NamedTuple):
     port: int
     fingerprint: Optional[str]
 
+
 class ExaConnection(object):
+    """
+    Warning:
+        Threads may share the module, but not connections
+        One connection may be used by different threads, just not at the same time
+        :meth:`pyexasol.ExaConnection.abort_query` is an exception,
+        it is meant to be called from another thread
+
+    Note:
+
+        It is advisable to use multiprocessing instead of threading and create
```

Comment:
```suggestion
        It is advisable to use multiprocessing instead of multithreading and create
```

Reply:
I think in this case `threading` makes more sense as there are no real "parallel" threads scheduled by the cpu in python.